components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/SynapseArtifactsApi.java

```diff
@@ -0,0 +1,49 @@
+package org.wso2.carbon.apimgt.internal.service;
+
+import org.wso2.carbon.apimgt.internal.service.dto.ErrorDTO;
+import org.wso2.carbon.apimgt.internal.service.SynapseArtifactsApiService;
+import org.wso2.carbon.apimgt.internal.service.impl.SynapseArtifactsApiServiceImpl;
+import org.wso2.carbon.apimgt.api.APIManagementException;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+import javax.inject.Inject;
+
+import io.swagger.annotations.*;
+import java.io.InputStream;
+
+import org.apache.cxf.jaxrs.ext.MessageContext;
+import org.apache.cxf.jaxrs.ext.multipart.Attachment;
+import org.apache.cxf.jaxrs.ext.multipart.Multipart;
+
+import java.util.Map;
+import java.util.List;
+import javax.validation.constraints.*;
+@Path("/synapse-artifacts")
+
+@Api(description = "the synapse-artifacts API")
+
+@Produces({ "application/json" })
+
+
+public class SynapseArtifactsApi  {
+
+  @Context MessageContext securityContext;
+
+SynapseArtifactsApiService delegate = new SynapseArtifactsApiServiceImpl();
+
+
+    @GET
+    
```

Comment:
Can we get rid of these additional new lines in the middle

Reply:
This is an auto generated file. We don't need to modify this.